docs/ibm-cloud-rules.md

```diff
@@ -3232,11 +3232,14 @@ components:
 </tr>
 <tr>
 <td valign=top><b>Description:</b></td>
-<td>Avoid using the same property name for properties of different types.</td>
+<td>Avoid using the same property name for properties of different types.
```

Comment:
Should this have a closing tag?

```suggestion
<td>Avoid using the same property name for properties of different types.</td>
```

Reply:
Actually, the `</td>` is a few lines later after the new sentence I added :)